examples/toolkits/generate_openai_tool_schema.py

```diff
@@ -0,0 +1,70 @@
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+
+import os
+
+from camel.agents import ChatAgent
+from camel.configs import ChatGPTConfig
+from camel.messages import BaseMessage
+from camel.models import ModelFactory
+from camel.toolkits import FunctionTool
+from camel.types import ModelPlatformType, ModelType
+
+# Set OpenAI API key
+api_key = os.getenv("OPENAI_API_KEY")
+if not api_key:
+    raise ValueError("API key not found in environment variables.")
+
+
+# Define a function which does't have a docstring
+def get_perfect_square(n: int) -> int:
+    return n**2
+
+
+# Create a model instance
+model_config_dict = ChatGPTConfig(temperature=1.0).as_dict()
+agent_model = ModelFactory.create(
+    model_platform=ModelPlatformType.OPENAI,
+    model_type=ModelType.GPT_4O_MINI,
+    model_config_dict=model_config_dict,
+)
```

Comment:
Yes, setting a lower temperature value would make it more deterministic. Do you think it would be better for me to simply remove the model setting and use the default model, or should I manually set temperature to a lower value?

Reply:
I think you can simply remove the model setting, as by default the `temperature` is `0.2`